components/speedreader/speedreader_throttle_unittest.cc

```diff
@@ -27,6 +28,11 @@ namespace {
 constexpr char kTestProfileName[] = "TestProfile";
 }  // anonymous namespace
 
+class TestSpeedreaderResultDelegate : public SpeedreaderResultDelegate {
```

Comment:
also it can go the anonymous namespace above

Reply:
done